dkg/dkg.go

```diff
@@ -136,24 +148,128 @@ func Run(ctx context.Context, conf Config) error {
 		return err
 	}
 
+	pubkeyToVerifiers := make(map[core.PubKey]*sharing.FeldmanVerifier)
```

Comment:
why not pass in shares directly into aggSignLock hash, no need for these verbose types I think

Reply:
i did this because we are getting response from exchange according to pubkeys and in aggSignLockhash also we are iterating according to pubkeys